pages/common/7z.md

```diff
@@ -21,7 +21,7 @@
 
 - Archive using a specific archive type:
 
-`7z a -t {{zip|gzip|bzip2|tar}} {{archived.7z}} {{path/to/file}}`
+`7z a -t {{zip|gzip|bzip2|tar}} {{archived.7z}} {{path/to/file_or_directory}}`
```

Comment:
This does not say "file or directory" in the description. Is this right ?

Reply:
Yes, it is right but not strictly following the third point: if you look at the full diff for that particular file, you see that the first example mentions "file or folder". Since 7z works with both files and folders, as a consequence I've also enforced the rule for other examples to make them consistent with the others :slightly_smiling_face: